File: app/src/main/java/com/kondee/wakeat/LocationModel.java
package com.kondee.wakeat;

import com.google.android.gms.maps.model.LatLngBounds;

import org.parceler.Parcel;

@Parcel
public class LocationModel {

    private LatLngBounds latLngBounds;

    public LatLngBounds getLatLngBounds() {
        return latLngBounds;
    }

    public LocationModel setLatLngBounds(LatLngBounds latLngBounds) {
        this.latLngBounds = latLngBounds;
        return this;
    }
}

File: app/src/main/java/com/kondee/wakeat/LocationModel.kt
package com.kondee.wakeat

import android.os.Parcelable
import com.google.android.gms.maps.model.LatLngBounds
import kotlinx.parcelize.Parcelize

@Parcelize
data class LocationModel(
    var latLngBounds: LatLngBounds? = null
) : Parcelable {

    fun setLatLngBounds(latLngBounds: LatLngBounds?): LocationModel {
        this.latLngBounds = latLngBounds
        return this
    }
}
